packages/mobile/src/common/types/localization/loclization.d.ts

```diff
@@ -0,0 +1,8 @@
+import 'react-i18next';
+import { resources } from '~/localization/i18n';
```

Comment:
```suggestion
import { resources } from './i18n';
```

Reply:
can't change, issues will hints won't work, and useTranslation get recursion depth issue